src/go/k8s/helm-chart/charts/ci/ct.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+charts:
+  - redpanda-operator
+target-branch: main
```

Comment:
shouldn't this be dev?

Reply:
also does this mean that the tests run against dev, not changes in the PR?

Reply:
The `ct lint` is only invoked by the CI. The `ct install` will be added in a future PRs. 

The target branch should be set to the branch that end user will install. I was thinking that releases are tagged from main branch.

Reply:
Thank you for that. I got wrong assumption that we tag the main branch. Will change that in next push.